js/index.js

```diff
@@ -1,7 +1,205 @@
 // Iteration 1: Names and Input
 // 
 
+
+let hacker1 = "Johnson";
+
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2 = "John";
+
+console.log(`The navigator's name is ${hacker2}`);
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+} else if (hacker2.length > hacker1.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`);
+} else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters`)
+}
+
+
+// Iteration 3: Loops
+
+function printInCapitalsWithSpaces(name) {
+    let nameInUpperCase = name.toUpperCase();
+    let newString = "";
+    for (let i = 0; i < name.length; i++) {
+        if (name[i+1]) {
+            newString += `${nameInUpperCase[i]} `; 
+        } else {
+            newString +=`${nameInUpperCase[i]}`;
+        }
+    }
+    console.log("New string is: ", newString);
+    return newString;
```

Comment:
Great solution! Very good that you wrote a function for it. (Instead of just writing the for loop etc.) I also like that you give the functions a meaningful name.